packages/expo-camera/ios/EXCamera/EXCamera.m

```diff
@@ -715,13 +716,11 @@ - (void)startSession
     [self onMountingError:@{@"message": @"Camera permissions not granted - component could not be rendered."}];
     return;
   }
-  EX_WEAKIFY(self);
-  dispatch_async(_sessionQueue, ^{
-    EX_ENSURE_STRONGIFY(self);
-
+  dispatch_async(self.sessionQueue, ^{
```

Comment:
I'm afraid that we can't just avoid weakifying – this may lead to some other issues that may come up at some later point (including memory leaks).

Reply:
Not sure why we need it here. The only thing that can happen is we extend the life of self. The queue will not retain the blocks. Just run through them until it's empty.

Reply:
Maybe objc works differently. I can put them back